### ISSUE TITLE
UIApplication applicationState being called from a Background Thread

### ISSUE THREAD

**Issue by LeonardoCardoso** [QuickBlox/quickblox-ios-sdk#935]
**New Issue Checklist**

 - [x] I have read the [Documentation](https://quickblox.com/developers/IOS)
 - [x] I have searched for a similar issue in the [project](https://github.com/QuickBlox/quickblox-ios-sdk/issues) and found none

**Environment details**

Info                           | Value                                       |
-------------------------------|---------------------------------------------|
iOS Version                    | 11.2.1                                    |                                          
Quickblox iOS SDK version      | 2.14                                 |
QuickbloxWebRTC SDK version    | 2.6.3                                  |                               
Xcode Version                  | Xcode 9.2                              |
Repro with our demo sample     | Yes   |

**Did this work before?**
Nope.

**Expected behavior**
-[UIApplication applicationState] must be used from main thread only

**Actual behavior**
runtime: UI API called from background thread: -[UIApplication applicationState] must be used from main thread only

**Logs**
```
Main Thread Checker: UI API called on a background thread: -[UIApplication applicationState]
PID: 3495, TID: 320482, Thread name: (none), Queue name: EncoderQueue, QoS: 0
Backtrace:
4   QuickbloxWebRTC                     0x0000000104479940 RTCIsUIApplicationActive + 56
5   QuickbloxWebRTC                     0x00000001044766ac _ZNSt3__119__tree_right_rotateIPNS_16__tree_node_baseIPvEEEEvT_ + 814372
6   QuickbloxWebRTC                     0x0000000104579ee4 RTCIsUIApplicationActive + 1050076
7   QuickbloxWebRTC                     0x0000000104688f4c RTCIsUIApplicationActive + 2160196
8   QuickbloxWebRTC                     0x0000000104698bac RTCIsUIApplicationActive + 2224804
9   QuickbloxWebRTC                     0x0000000104680da0 RTCIsUIApplicationActive + 2127000
10  QuickbloxWebRTC                     0x0000000104682624 RTCIsUIApplicationActive + 2133276
11  QuickbloxWebRTC                     0x00000001044742a4 _ZNSt3__119__tree_right_rotateIPNS_16__tree_node_baseIPvEEEEvT_ + 805148
12  libdispatch.dylib                   0x0000000106e6145c _dispatch_client_callout + 16
13  libdispatch.dylib                   0x0000000106e70110 _dispatch_queue_serial_drain + 692
14  libdispatch.dylib                   0x0000000106e649a4 _dispatch_queue_invoke + 332
15  libdispatch.dylib                   0x0000000106e71104 _dispatch_root_queue_drain_deferred_wlh + 424
16  libdispatch.dylib                   0x0000000106e78100 _dispatch_workloop_worker_thread + 652
17  libsystem_pthread.dylib             0x000000018490afd0 _pthread_wqthread + 932
18  libsystem_pthread.dylib             0x000000018490ac20 start_wqthread + 4
```

**Comment by Pro100Andrey** [comment]
Hi @LeonardoCardoso, Thanks for reporting the issue!

**Comment by Raikerian** [comment]
This will be fixed in a near future release. For now, this is just a warning from thread checker.

**Comment by Pro100Andrey** [comment]
Hi @LeonardoCardoso, fixed (v2.7 - Mar 28, 2018)

[Changelog](https://quickblox.com/developers/Sample-webrtc-ios#Framework_changelog)

Thanks!

**Comment by fukemy** [comment]
hello, im really sorry for bother you, but im stuck in same problem with RTCFramework swift:
```

PID: 2257, TID: 789400, Thread name: DecodingThread, Queue name: com.apple.root.default-qos.overcommit, QoS: 0
Backtrace:
4   WebRTC                              0x0000000104935ea0 RTCIsUIApplicationActive + 56
5   WebRTC                              0x0000000104930f60 RTCSetVertexData + 90988
6   WebRTC                              0x0000000104a58e20 RTCIsUIApplicationActive + 1191864
7   WebRTC                              0x0000000104c85fec RTCIsUIApplicationActive + 3473796
8   WebRTC                              0x0000000104c952a0 RTCIsUIApplicationActive + 3535928
9   WebRTC                              0x0000000104c953a0 RTCIsUIApplicationActive + 3536184
10  WebRTC                              0x0000000104c78684 RTCIsUIApplicationActive + 3418140
11  WebRTC                              0x0000000104960d04 RTCIsUIApplicationActive + 175772
12  WebRTC                              0x0000000104960b98 RTCIsUIApplicationActive + 175408
13  libsystem_pthread.dylib             0x000000019f5bd2c0 <redacted> + 128
14  libsystem_pthread.dylib             0x000000019f5bd220 _pthread_start + 44
15  libsystem_pthread.dylib             0x000000019f5c0cdc thread_start + 4
2019-12-16 09:55:43.009564+0700 CiviX_IOS[2257:789400] [reports] Main Thread Checker: UI API called on a background thread: -[UIApplication applicationState]
PID: 2257, TID: 789400, Thread name: DecodingThread, Queue name: com.apple.root.default-qos.overcommit, QoS: 0
Backtrace:
4   WebRTC                              0x0000000104935ea0 RTCIsUIApplicationActive + 56
5   WebRTC                              0x0000000104930f60 RTCSetVertexData + 90988
6   WebRTC                              0x0000000104a58e20 RTCIsUIApplicationActive + 1191864
7   WebRTC                              0x0000000104c85fec RTCIsUIApplicationActive + 3473796
8   WebRTC                              0x0000000104c952a0 RTCIsUIApplicationActive + 3535928
9   WebRTC                              0x0000000104c953a0 RTCIsUIApplicationActive + 3536184
10  WebRTC                              0x0000000104c78684 RTCIsUIApplicationActive + 3418140
11  WebRTC                              0x0000000104960d04 RTCIsUIApplicationActive + 175772
12  WebRTC                              0x0000000104960b98 RTCIsUIApplicationActive + 175408
13  libsystem_pthread.dylib             0x000000019f5bd2c0 <redacted> + 128
14  libsystem_pthread.dylib             0x000000019f5bd220 _pthread_start + 44
15  libsystem_pthread.dylib             0x000000019f5c0cdc thread_start + 4`
```
So can you provide the way how to fix this problem as you did? Many thanks
